projects/canopy/src/lib/data-point/docs/guide.stories.mdx

```diff
@@ -0,0 +1,38 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import Feedback from '../../docs/common/feedback.md';
+
+<Meta title="Components/Data point/Guide" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# Data point
+
+<p class="standfirst">Display data organised by heading and value. A data point can be used on its own, or grouped into a data point list where each one will be displayed horizontally and given a list-item role.</p>
+
+## Usage
+
+Import the data point module in your application:
+
+```js
+@NgModule({
+  ...
+  imports: [ ..., LgDataPointModule ],
+});
+```
+
+In your HTML, for a single data point:
+
+<Source id="components-data-point-examples--single-data-point"></Source>
+
+and for a data point list:
+
+<Source id="components-data-point-examples--data-points-list-story"></Source>
+
+### Inputs
+
+| Name           | Description                        |   Type   | Default | Required |
+|----------------|------------------------------------|:--------:|:-------:|:--------:|
+| `headingLevel` | Allows the heading level to be set | `number` |   n/a   |   Yes    |
+
+---
```

Comment:
Is this line intentional?

Reply:
Yes, we separate the main content from the feedback 👍